dataactbroker/handlers/jobHandler.py

```diff
@@ -14,13 +14,13 @@ class JobHandler(JobTrackerInterface):
     engine -- sqlalchemy engine for generating connections and sessions
     connection -- sqlalchemy connection for executing direct SQL statements
     session -- sqlalchemy session for ORM usage
-    waitingStatus -- status_id for "waiting"
-    runningStatus -- status_id for "running"
-    finishedStatus -- status_id for "finished"
-    fileUploadType -- type_id for "file_upload"
-    dbUploadType -- type_id for "db_upload"
-    validationType -- type_id for "validation"
-    externalValidationType -- type_id for "external_validation"
+    waitingStatus -- job_status_id for "waiting"
```

Comment:
So most of these no longer exist as instance variables, I'll push a commit to remove these lines.